docs/policy.md

```diff
@@ -0,0 +1,169 @@
+# Policy file explained
+
+Say we generate a policy file for a project with the following dependency tree:
+```
+app.js
+└─┬ some-package
+  └─┬ entropoetry
+    └── bn.js
+```
+
+`./lavamoat/node/policy.json`:
+```
+{
+  "resources": {
+    "some-package": {
+      "globals": {
+        "Buffer.from": true
+      },
+      "packages": {
+        "some-package>entropoetry": true
+      }
+    },
+    "some-package>entropoetry": {
+      "builtin": {
+        "assert": true,
+        "buffer.Buffer": true,
+        "zlib": true
+      },
+      "globals": {
+        "console": true,
+        "process.exitCode": "write",
+      },
+      "packages": {
+        "some-package>entropoetry>bn.js": true
+      }
+    },
+    "some-package>entropoetry>bn.js": {
+      "builtin": {
+        "buffer.Buffer": true
+      },
+      "globals": {
+        "Buffer": true
+      }
+    }
+  }
+}
+```
+
+## Fields
+### resources 
+
+All packages in your dependency graph accessible via `require()`.  
```

Comment:
"LavaMoat picks the shortest dependency path for each entry in resources": This sounds like LavaMoat will go through each package in `resources` and choose another name for each entry. But this doesn't seem right. Perhaps replace this section with:

```
All packages in your dependency graph accessible via `require()`. The keys of this object are names of packages assigned by LavaMoat. In order to identify a package, LavaMoat does not simply use the published name of the package, but generates a unique name by tracing the package's shortest route through its dependents within the dependency tree. This is done in order to prevent a malicious package which shares the same name as a package published to NPM from hijacking permissions.
```

Reply:
I'll take that